examples/experimental/i3s-17-and-debug/components/debug-panel.js

```diff
@@ -114,6 +116,13 @@ export default class DebugPanel extends PureComponent {
     });
   }
 
+  toggleMinimapViewport() {
+    const {minimapViewport} = this.state;
+    this.setState({minimapViewport: !minimapViewport}, () => {
+      this.applyOptions();
```

Comment:
Add comment explaining why this needs to be done async.

Reply:
It is async in every panel handler. As soon as `applyOptions` takes options from state, the state should be actual.